src/main/scala/eu/proteus/solma/fd/FrequentDirections.scala

```diff
@@ -89,10 +89,10 @@ object FrequentDirections {
   implicit def fitNoOp[T] = {
     new StreamFitOperation[FrequentDirections, T]{
       override def fit(
-          instance: FrequentDirections,
```

Comment:
I did make changes locally, and then reverted them back after the discussion about the fit for datasets. I will revert those changes so it matches the original version.